recipe-client-addon/lib/Storage.jsm

```diff
@@ -29,6 +30,34 @@ function loadStorage() {
 }
 
 this.Storage = {
+  DURABILITY_NAMESPACE: '_storageDurability',
+  DURABILITY_KEY: 'durable',
+  async seedDurability(sandbox) {
+    const globalDurabilityStore = Storage.makeStorage(this.DURABILITY_NAMESPACE, sandbox);
+    let durability = await globalDurabilityStore.getItem(this.DURABILITY_KEY);
+
+    durability = parseInt(durability, 10);
```

Comment:
The storage driver [stores simple values directly](http://normandy.readthedocs.io/en/latest/dev/driver.html#Storage) instead of converting them to strings like localstorage would. Can we assume it returns numbers instead of doing the parseInt/isNaN dance?